pages/cql/OpioidCDS_DSTU2_REC_10.cql

```diff
@@ -55,15 +55,14 @@ define "Validate Trigger Event":
   )
 
 define "Inclusion Criteria":
-  AgeInYears() >= 18
+  AgeInYearsAt(Now()) >= 18
     and "Validate Trigger Event"
     and not "Exclusion Criteria"
     and AnyTrue(Common.ProbableDaysInRange(Common."Active Ambulatory Opioid Rx", 90, 80))
     and
       ("No Urine Screening In Last 12 Months"
-        or "Has Missing Opioids?"
-        or "Has Unprescribed Opioids?"
-        or "Has Illicit Drugs in Screening?")
+        or "Has Evidence of Unprescribed Opioids"
```

Comment:
This should just be "Has Evidence of Opioids", we don't have any basis for saying whether they are prescribed or not.

Reply:
I have removed the notion of "prescribed".